Example/WatchDemo/WatchDemo/WatchDemoApp.swift

```diff
@@ -14,6 +14,7 @@
 
 import Firebase
 import SwiftUI
+import FirebaseCore
```

Comment:
Do you need this import? It should be included with `import Firebase`

Reply:
Oops remove this.